Revised (#131)
greenelab/iscb-diversity-manuscript: content/20.results.md

In version 3, instead of weighting all articles equally as in the earlier versions, we used citation count to weight articles to control for the differential impact of research contributions.
Using citation counts has key limitations: female scientists of different disciplines are cited less often than their male counterparts [@arxiv:2001.01002; @doi:10.1002/ece3.4993; @doi:10.1038/s41550-017-0141].
Furthermore, the act of being honored, particularly with a keynote at an international meeting, could lead to work being more recognized and cited, which would reverse the arrow of causality.

> We expected this approach to be overly conservative, leading to a fail to reject the null hypothesis of no disparities even if disparities existed.

Reviewer: ```suggestion
We expected this approach to be overly conservative, leading to a failure to reject the null hypothesis of no disparities even if disparities existed.
```

](https://raw.githubusercontent.com/greenelab/iscb-diversity/ac43b098a2a994cbe3788bf7a19f541fc533b3de/figs/region_breakdown.png){#fig:region_breakdown}

> An alternative approach that considered each pair of forename and last name as the unit of measurement showed similar results.

Reviewer: ```suggestion
Consistent disparities were also apparent with an alternative approach that considered each pair of forename and last name as the unit of measurement.
```

> In addition, if we assume that honors accrue 10 years after their most prolific year with respect to authorships,

Reviewer: ```suggestion
In addition, a time-lagged model, which might be appropriate if we assume that honors accrue ten years after an authors most prolifically cited year, 
```

> a 10-year lag model results in a similar underrepresentation of East Asian scientists' names in the group of honorees, though the effect size is less striking.

Reviewer: ```suggestion
results in a similar underrepresentation of East Asian scientists' names in the group of honorees, though the effect size is smaller.
```

For example, the proportion of honor associated with East Asian name origins in 2019 is still substantially less than the proportion of senior authorships associated with East Asian names in 2009.

> Please see the [analysis notebook](https://greenelab.github.io/iscb-diversity/11.name-origin.html) for more details.

Reviewer: ```suggestion
The [analysis notebook](https://greenelab.github.io/iscb-diversity/11.name-origin.html) for this portion provides the results in these scenarios.
```

> When applying the Wiki2019-LSTM model to the name origins of only US-affiliated scientists, we found a similar underrepresentation of honors to scientists with East Asian names (OR~East\ Asian~ = 0.17, $\beta$~East\ Asian~ = -1.77, _p_ = 4.3×10^-4^).

Reviewer: ```suggestion
We sought to disentangle geography from other factors by examining results for the country with the most affiliated scientists receiving honors, the US.
When applying the Wiki2019-LSTM model to the name origins of only US-affiliated scientists, we found a similar underrepresentation of honors to scientists with East Asian names (OR~East\ Asian~ = 0.17, $\beta$~East\ Asian~ = -1.77, _p_ = 4.3×10^-4^).
```

](https://raw.githubusercontent.com/greenelab/iscb-diversity/ac43b098a2a994cbe3788bf7a19f541fc533b3de/figs/region_breakdown.png){#fig:region_breakdown}

When applying the Wiki2019-LSTM model to the name origins of only US-affiliated scientists, we found a similar underrepresentation of honors to scientists with East Asian names (OR~East\ Asian~ = 0.17, $\beta$~East\ Asian~ = -1.77, _p_ = 4.3×10^-4^).
We observed no statistically significant difference between the proportion of honors given to authors with Celtic/English names, European names, or names in Other categories (_p_ = 0.60, _p_ = 0.11, and _p_ = 0.70, respectively).
Please see the [US analysis notebook](https://greenelab.github.io/iscb-diversity/14.us-name-origin.html) for more details.

> An alternative approach that considered each pair of forename and last name as the unit of measurement showed similar results.

Reviewer: We will need to write an extremely short methods paragraph about this. Also, we should probably expand the considerations paragraph to discuss why we don't use this strategy as the primary approach:
* selecting an honoree a second time really is a distinct selection process
* name collisions are hazardous to analyses and tools like ORCID are not widespread enough to be used for this purpose yet.

In addition, if we assume that honors accrue 10 years after their most prolific year with respect to authorships,
a 10-year lag model results in a similar underrepresentation of East Asian scientists' names in the group of honorees, though the effect size is less striking.
For example, the proportion of honor associated with East Asian name origins in 2019 is still substantially less than the proportion of senior authorships associated with East Asian names in 2009.

> Please see the [analysis notebook](https://greenelab.github.io/iscb-diversity/11.name-origin.html) for more details.

Reviewer: One more thing - I wonder if we should do this before we dive into the US-affiliated authors. That is a deeper analysis, so we're ending the section on a weaker setup that we think is flawed instead of the stronger one.

Reviewer 2: Good point. I'll rearrange.
